### DISCUSSION TITLE
怎么三天两头有人要在群里买个机器人的

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#494] [General]
是有人拿去盈利了？ @SilveryStar 你那边知道么？

**Comment by SilveryStar** [comment]
不清楚，我那边倒是来了个卖自抽号的，但也没在频道里说
（MIT 的就是有算卖也没法管吧

**Reply by Arondight** [reply]
确实是这样的，我简单在 readme 里提了一下。我的 qq 和 bot 的没有共同的群聊，bot 本身也没加多少群，所以也不知道是啥情况。这两天收到好几个带话说想买一个的，我也不清楚是有人做这个服务了还是大家下意识的要花钱

**Comment by mark9804** [comment]
我猜测是因为有其他和我们项目无关的机器人，那些机器人是收费的，所以可能有用户以为我们的机器人也需要收费，希望真实情况就是这样

> MIT 的就是有算卖也没法管吧

如果真有拿这两个仓库的代码去盈利的行为，就算 bot 的协议一开始是 GPL 也没法管，基本上只能抵制一下。真有人想恰这笔钱凭个人开发者的资源是拦不住的
